aesthetic_ants/penguin/game.py

```diff
@@ -11,6 +11,7 @@
 class Game(pyglet.window.Window):
     def __init__(self, *, fps=120, **kwargs):
         super().__init__(caption='Penguin Snowball', **kwargs)
+        self.is_over = False
```

Comment:
The game should use this attribute to stop the game from updating, as it still updates the enemies and player when the game is over.